### PR TITLE
docs: fix trailing period

### PR DESCRIPTION
**Pull request by dennisme**
**What this PR does / why we need it**:

Noticed this when checking out the docs here: https://kubernetes-sigs.github.io/cluster-api/getting_started/existing_providers.html. 

This moved the period up a line.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

n/a

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
fix trailing period in getting started docs
```

### CODE REVIEW

**Comment by dennisme** [comment]
I should be all set on the CLA, but if not I can double check.

File: docs/book/getting_started/existing_providers.md [hunk 1]
```diff
@@ -5,5 +5,4 @@ There are multiple implementations of the Cluster API for different environments
 These implementations are called _providers_. To find a partial list of the 
 existing providers see the Cluster API [Provider Implementations List][provider-implementations-list]
 
-[provider-implementations-list]: https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md#provider-implementations
-.
+[provider-implementations-list]: https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md#provider-implementations.
```
Comment (davidewatson) [inline docs/book/getting_started/existing_providers.md]: This is a definition. The missing period is in the line above where the link is referenced.

**Pull request merged**